### PR TITLE
Spring

### PR DESCRIPTION
**Pull request by StVit1390**
spring fork for fadeIn effect to sections

### CODE REVIEW

File: Landing/app/page.tsx [hunk 1]
```diff
@@ -77,14 +80,14 @@ export default function Home() {
             <Header />
             {burger && <MobileMenu />}
             {!burger && <>
-              <FirstSection />
```
Comment (hubochkin) [inline Landing/app/page.tsx:80]: const sections = [
  FirstSection,
  SecondSection,
  ThirdSection,
  FourthSection,
  FifthSection,
  SixthSection,
  SeventhSection,
  EighthSection
];
we can do something like this
{!burger && sections.map((SectionComponent, index) => (
  <FadeInSection key={index}>
    <SectionComponent />
  </FadeInSection>
))}

File: Landing/components/eighthSection/styles.ts [hunk 2]
```diff
@@ -77,7 +76,10 @@ const fadeIn = keyframes`
 export const CustomField = styled(TextField)`
     background-color: ${({theme})=> theme.palette.background.default} ;
     border-radius: 13px;
+<<<<<<< HEAD
```
Comment (hubochkin) [inline Landing/components/eighthSection/styles.ts:79]: It will not compile because we have this HEAD

File: Landing/elements/textCutter/index.tsx [hunk 3]
```diff
@@ -0,0 +1,68 @@
...
+        }
+    }, [children])
+
+    const { local } = useContext(LocalContext)
+    
+    const ButtonLocale = () => {
+        let btnName 
+
+        switch(local){
+            case 'en': btnName = "read more";
```
Comment (hubochkin) [inline Landing/elements/textCutter/index.tsx:31]: WE can create a enum with that and use enum here

File: Landing/app/page.tsx (hunk 1 above)
Comment (StVit1390) [reply Landing/app/page.tsx:80]: done

File: Landing/components/eighthSection/styles.ts (hunk 2 above)
Comment (StVit1390) [reply Landing/components/eighthSection/styles.ts]: done

File: Landing/elements/textCutter/index.tsx (hunk 3 above)
Comment (StVit1390) [reply Landing/elements/textCutter/index.tsx]: done but i dont get why is it better to use enum

**Comment by hubochkin** [comment]
@StVit1390  Resolve conflicts before merge please

**Pull request merged**